apps/landingPage/components/Header.js

```diff
@@ -67,6 +67,10 @@ const GithubLink = styled.div`
   }
 `;
 
+const LinkWithHeight = styled.div`
+  height: 42px;
```

Comment:
I was just thinking. What about use 44px? We could get it from `defaultTokens` (heightButtonNormal) and actually 44pt (~px) is some kind of factory standard for minimal touchable size for tablets and mobile devices (not for PC).

Apple also recommends it:
<img width="815" alt="screenshot 2019-03-08 at 17 03 28" src="https://user-images.githubusercontent.com/858321/54039881-4e9fc600-41c4-11e9-9034-68e261778cbe.png">

https://developer.apple.com/design/human-interface-guidelines/ios/visual-design/adaptivity-and-layout/

Reply:
Thats a good point but I m not sure in this case.
I put height here only because somehow wrapping `img` into `a` was increasing height of the `a` more than the `img` height so I couldnt align logo properly in the mobile header

But still, as for the 44px number, I am not sure what was the height intended in design so I just took 42px number by eye

Reply:
I think that 42px is OK because the width is much more. Maby and some point we will focus on accessibility and improve it.

Reply:
Alright, keeping 42px for now then